spec/spec_helper.rb

```diff
@@ -1,5 +1,9 @@
+require 'simplecov'
+SimpleCov.start
+
 $LOAD_PATH.unshift File.expand_path("../../", __FILE__)
 require "rspec-console"
```

Comment:
you added "require pry" IIRC.
so instead of requiring all your gemfile, at some point, you just want to use bundler. I don't feel strongly about this.